src/shared/orgUtils.ts

```diff
@@ -38,4 +38,20 @@ export class OrgUtils {
 
     return undefined;
   }
+
+  /**
+   * Checks to see if Local Dev is enabled for the org.
+   *
+   * @param connection the connection to the org
+   * @returns boolean indicating whether Local Dev is enabled for the org.
+   */
+  public static async isLocalDevEnabled(connection: Connection): Promise<boolean> {
+    const metadata = await connection.metadata.read('LightningExperienceSettings', 'enableLightningPreviewPref');
+    // casting to any here b/c LightningExperienceSettings type which is defined in 'jsforce'
```

Comment:
For what it's worth, this is something that TypeScript is well suited to solving with types. As an example:

```typescript
type LightningPreviewMetadataResponse = {
  enableLightningPreviewPref: boolean | undefined;
};

// ...

const flagValue = (metadata as LightningPreviewMetadataResponse).enableLightningPreviewPref;
const localDevEnabled = flagValue ?? false;
return localDevEnabled;
```

It may need some adjustment based on the actual data you get back, but this is the preferred approach over casting to `any`, which is why there's linting violation to push you in this direction.

Reply:
Oh good point, I'll update the code. For some reason it is actually returned back as a `string` and not a `boolean` but regardless, your proposed approach is better than what I've been doing.